scripts/install_run_precommit.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+# shellcheck disable=SC1091
+
+python3 -m venv .venv
```

Comment:
Should we add small doc in `docs/`?

Reply:
Writing is hard. I put it off until we'd agreed this was a promising direction. I'll add a doc now.